com.unity.shadergraph/Documentation~/Sample-Texture-2D-Array-Node.md

```diff
@@ -16,6 +16,10 @@ NOTE: This [Node](Node.md) can only be used in the **Fragment** shader stage.
 | Index   | Input | Float       | None  | Index of array to sample |
 | UV      | Input | Vector 2    |   UV  | UV coordinates |
 | Sampler | Input | Sampler State | Default sampler state | Sampler for the texture |
+| LOD | Input | Float | LOD | Available when mip sampling mode is LOD. The actual mip to sample from. |
+| Bias | Input | Float | Bias | Available when mip sampling mode is Bias. A bias on top of the current global mip bias to sample from. |
```

Comment:
this applies to the rest of the docs as well, but there is any docs in the main unity manual that explain what mip levels are that we can cross link to?

Reply:
Sarah is working on this right now.

Reply:
Can confirm, have something in progress that I will share shortly :)

Reply:
Tests are green. I am ready to merge this.